docs/source/migration/4.0.mdx

```diff
@@ -426,7 +426,7 @@ import data.builders.schema.__Schema
 
 ### KotlinLabs directives are now version 0.2
```

Comment:
```suggestion
### KotlinLabs directives are now version 0.3
```

Reply:
Actually, this needs to be in the `next-docs` branch

Reply:
Woops good call! 👍